docs/asdf/features.rst

```diff
@@ -413,6 +413,15 @@ details).
 Saving ASDF in FITS
 ===================
 
+.. note::
+
+    `~asdf.fits_embed.AsdfInFits` and `~asdf.fits_embed` are deprecated and
+    will be removed from an upcoming version of ASDF. Support for reading
+    and writing ASDF data within a FITS file has moved to the `stdatamodels`
+    package. Please see the the `stdatamodels`
+    :external+stdatamodels:doc:`asdf_in_fits` documentation for information
```

Comment:
This should be an intersphinx link. Is this the issue the intersphinx link that will eventually be resolved by spacetelescope/stdatamodels#122? If so we should create an issue to track updating this to a proper link when `stdatamodels` gets fixed.

Reply:
Thanks. I opened an issue here:
https://github.com/asdf-format/asdf/issues/1385